.github/workflows/markdown.yaml

```diff
@@ -51,32 +51,35 @@ on:
           **/*.MD
           #.github
           #vendor
+          #RELEASE_NOTES.md
 jobs:
   changes:
+    if: ${{ !github.event.pull_request.draft }}
     runs-on: ubuntu-latest
     outputs:
       files-changed: ${{ steps.filter.outputs.markdown }}
     steps:
       - uses: actions/checkout@v4
-      - uses: dorny/paths-filter@v2
+        with:
+          fetch-depth: 2 # To retrieve the preceding commit.
+      - uses: mirpedrol/paths-filter@changes-last-commit
```

Comment:
Isn't this feature available on the main repo? https://github.com/dorny/paths-filter#:~:text=%23%20Branch%2C%20tag,e.g.%20master)

Reply:
not for PRs :/

Reply:
If the aim here is to only check the previous commit and not the full PR, does this mean that if I push 5 commits there will be no checks on the files modified in the first 4 commits?

Reply:
the aim is to not run checks if for a given action (e.g. markdown check) if no monitored file is changed. in case even only one file is changed, the action runs on all files. still I don't think this patch works correctly, so I will revert to official dorny, and see if I can modify it to what we need.